README.md

```diff
@@ -14,6 +14,14 @@ Compiled and printed versions of the book are available at the
 and nightly builds are available on the
 [github wiki](https://github.com/HoTT/book/wiki/Nightly-Builds).
 
+## Editing the book
+
+This book is not a community project, but we do welcome our readers to suggest minor improvements. The best way to propose an edit is to open a pull request with your suggested change.
+
+## Code of conduct
+
+For many, the HoTT book is their introduction to our subject and our diverse community, including people from any nationality, gender identity, sexual orientation, race, colour, ability, and background. In order to ensure for everyone a welcoming and inclusive environment in our discussions, we follow the guidelines of the [github code of conduct](https://docs.github.com/en/site-policy/github-terms/github-community-forum-code-of-conduct). You can expect from the authors and any participant that we are kind and respectful in discussions, that we use inclusive language, and that we do our best to understand each other's different perspectives. It might not always be that we accept the change in the way you proposed it, but we always value your input, regardless of your level of experience or status within the community.
```

Comment:
This is fairly large refactoring, but I think it's best to explicitly include what we care about from the github CoC (and perhaps cite it) rather than to invoke it by name. It's far too legalistic to simply #include, IMO. See, for example, how the [django coc](https://www.djangoproject.com/conduct/) evolved from the SpeakUp! coc.

FWIW I also think basing our CoC on one like Django's or Speakup's would be better as they are more focused on maintaining a healthy community, while GH's is mostly cover-your-ass. cf [this comment](https://github.com/HoTT/book/pull/1101#issuecomment-1096447140) for more on this.

Reply:
@andrejbauer @atondwal @EgbertRijke I wonder if it makes more sense to have a different PR? I think we need a protocol for people to propose different CoCs, refine each of them separately, discuss the differences among them, and eventually pick one. Multiple PRs might be a way, and I wonder whether there's a better one. In any case, I am looking forward to the version proposed by @atondwal.